### PR TITLE
Database setup for test layers

### PR DESCRIPTION
**Pull request by bhasin85**

### CODE REVIEW

File: src/main/src/jetty/database/gogoduck-testing/cars.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO cars_timeseries_url VALUES ('CSIRO/Climatology/CARS/2009/eMII-product/CARS2009_World_monthly.nc','2016-04-27 02:29:34');
+INSERT INTO cars_timeseries_url VALUES ('CSIRO/Climatology/CARS/2009/eMII-product/CARS2009_Australia_weekly.nc','2016-04-27 02:29:34');
```
Comment (jonescc) [inline src/main/src/jetty/database/gogoduck-testing/cars.sql]: Are we missing layers for these?  Need one for each file here.

File: src/main/src/jetty/database/gogoduck-testing/srs.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO srs_timeseries_url VALUES ('IMOS/SRS/SST/ghrsst/L3S-1d/ngt/2016/20160712152000-ABOM-L3S_GHRSST-SSTskin-AVHRR_D-1d_night.nc','2016-07-16 07:50:28');
+INSERT INTO srs_timeseries_url VALUES ('IMOS/SRS/SST/ghrsst/L3S-1d/ngt/2016/20160713152000-ABOM-L3S_GHRSST-SSTskin-AVHRR_D-1d_night.nc','2016-07-17 03:46:50');
+INSERT INTO srs_timeseries_url VALUES ('IMOS/SRS/SST/ghrsst/L3S-1d/ngt/2016/20160714152000-ABOM-L3S_GHRSST-SSTskin-AVHRR_D-1d_night.nc','2016-07-16 00:45:59');
```
Comment (jonescc) [inline src/main/src/jetty/database/gogoduck-testing/srs.sql]: Missing layer?

File: src/main/src/jetty/database/gogoduck-testing/srs_oc.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO srs_oc_timeseries_url VALUES ('IMOS/SRS/OC/gridded/contributed/SO-Johnson/chl/8d/seawifs/S20103452010352.L3m_8D_SO_Chl_9km.Johnson_SO_Chl.nc','2016-03-16 11:54:01');
+INSERT INTO srs_oc_timeseries_url VALUES ('IMOS/SRS/OC/gridded/contributed/SO-Johnson/chl/8d/seawifs/S20103372010344.L3m_8D_SO_Chl_9km.Johnson_SO_Chl.nc','2016-03-16 11:54:00');
+INSERT INTO srs_oc_timeseries_url VALUES ('IMOS/SRS/OC/gridded/contributed/SO-Johnson/chl/8d/seawifs/S20103292010336.L3m_8D_SO_Chl_9km.Johnson_SO_Chl.nc','2016-03-16 11:53:23');
```
Comment (jonescc) [inline src/main/src/jetty/database/gogoduck-testing/srs_oc.sql]: Missing layer?

File: src/main/src/jetty/database/gogoduck-testing/cars.sql (hunk 1 above)
Comment (jonescc) [inline src/main/src/jetty/database/gogoduck-testing/cars.sql]: Weekly and monthly aren't really aggregate-able - perhaps just use the weekly one or have separate layers for weekly and monthly  - the processing should be the same though so its probably not required.

File: src/main/src/jetty/geoserver_data/workspaces/imos/geoserver/datastore.xml [hunk 4]
```diff
@@ -8,17 +8,17 @@
     <id>WorkspaceInfoImpl--7f665eac:150d9d01f4d:-7ffe</id>
   </workspace>
   <connectionParameters>
-    <entry key="schema">geoserver</entry>
-    <entry key="Estimated extends">true</entry>
-    <entry key="fetch size">1000</entry>
-    <entry key="encode functions">false</entry>
+    <entry key="schema">public</entry>
```
Comment (jonescc) [inline src/main/src/jetty/geoserver_data/workspaces/imos/geoserver/datastore.xml]: This needs to be geoserver for this to work for me.

Comment (bhasin85) [inline src/main/src/jetty/geoserver_data/workspaces/imos/geoserver/datastore.xml]: @jonescc  Updated the schema to geoserver

**Pull request merged**